### PR TITLE
Brandon release

### PR DESCRIPTION
**Pull request by branflake2267**
prepping for release. changed the lib setup some.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +87,16 @@ This library provides Java language bindings for the Google maps Javascript api.
 	<inherits name='com.google.gwt.maps.Maps' />
 	```
 
+##Utility Library
+
+    ```xml
+    <dependency>
+        <groupId>com.github.branflake2267.gwt-maps-utility</groupId>
+        <artifactId>markerclustererplus</artifactId>
+        <version>2.0.9-build-1</version>
```
Comment (tvernum) [inline README.md:96]: Shouldn't this be 2.0.15-build-1 ?

Comment (branflake2267) [inline README.md:96]: Good catch. I have changed it.

**Comment by branflake2267** [comment]
Ugh, Seems I have an issue with the version in the pom so the build didn't go correct. I've been working late so everything is taking 10x as long. I'm going to give up in a few, and finish it tomorrow. Seems I need to get the new lib configured better. I was hoping to put it in a sub directory but it installed the source in a different directory.

**Comment by branflake2267** [comment]
alpha-6 released

**Pull request merged**